src/entities/purchase-flow-params.ts

```diff
@@ -0,0 +1,24 @@
+import { type Package, type PurchaseOption } from "./offerings";
+
+/**
+ * Parameters used to customise the purchase flow when invoking the `.purchase` method.
+ * @public
+ */
+export interface PurchaseFlowParams {
+  /**
+   * The package you want to purchase. Obtained from {@link Purchases.getOfferings}.
+   */
+  rcPackage: Package;
+  /**
+   * The option to be used for this purchase. If not specified the default one will be used.
+   */
+  purchaseOption?: PurchaseOption | null;
```

Comment:
not a JS expert, but wondering if this needs to be nullable as well, or just optional?

Reply:
Mentioned my logic here: https://github.com/RevenueCat/purchases-js/pull/117#discussion_r1625794328